cli/flags.rs

```diff
@@ -1183,6 +1183,7 @@ fn test_subcommand<'a, 'b>() -> App<'a, 'b> {
         .takes_value(true)
         .help("Run tests with this string or pattern in the test name"),
     )
+    .setting(clap::AppSettings::AllowLeadingHyphen)
```

Comment:
Maybe use https://docs.rs/clap/2.33.2/clap/enum.ArgSettings.html#variant.AllowLeadingHyphen on the `"filter"` arg instead of using `AppSetting`

Reply:
Oof! Youre right! Thank you